src/content/docs/en/guides/internationalization.mdx

```diff
@@ -307,6 +307,29 @@ export default defineConfig({
 
 Astro will ensure that a page is built in `src/pages/fr` for every page that exists in `src/pages/es/`. If the page does not already exist, then a page with a redirect to the corresponding `es` route will be created.
 
+### `fallbackType`
+
+<Since v="4.15.0" />
+
+The configuration `i18n.fallbackType` allows to change the behavior of the fallback strategy. By default, Astro will redirect to the fallback locale when a page is not found. You can change this behavior to show the content of the fallback locale without redirecting.
+
+When `i18n.fallbackType` is set to `"rewrite"`, Astro will show the content of the fallback locale without redirecting. This means that the URL will remain the same, but the content will be from the fallback locale.
+
+```js title="astro.config.mjs" ins={8}
+export default defineConfig({
+  i18n: {
+    defaultLocale: "en",
+    locales: ["es", "en", "fr"],
+    fallback: {
+      fr: "es"
+    },
+    fallbackType: "rewrite" // the value `"redirect"` is the default behaviour and value
```

Comment:
nit: we use both the spelling `behavior` and `behaviour` here, we should default to the American spelling:
```suggestion
    fallbackType: "rewrite" // the value `"redirect"` is the default behavior and value
```